### PR TITLE
fix: moved i2c_platform file to user agnostic directory

### PR DESCRIPTION
**Pull request by builderjer**
It was mentioned [here](https://github.com/OpenVoiceOS/ovos-i2csound/pull/9#discussion_r1586948189) that this only works for user `ovos`.  This moves the `i2c_platform` file to `/etc/mycroft/i2c_platform` to not hard code a user.

It was mentioned to set an environment file, but those can only be set for the current shell, or on a reboot.  System environment variables CAN NOT be set dynamically according to everything I have read.

### CODE REVIEW

File: ovos-i2csound [hunk 1]
```diff
@@ -73,10 +73,11 @@ compare_kernel_version() {
 main() {
     # Create a variable to hold the detected device name
     local i2c_device_name
+    local i2c_device_file=$HOME/.config/mycroft/i2c_platform
```
Comment (NeonDaniel) [inline ovos-i2csound]: Instead of `mycroft`, consider `${OVOS_CONFIG_BASE_FOLDER:-mycroft}` to match ovos-config directory handling

**Review by NeonDaniel — CHANGES_REQUESTED** [review]

File: ovos-i2csound (hunk 1 above)
Comment (JarbasAl) [reply ovos-i2csound]: this does not use a config nor is it part of ovos-core, IMHO it should use a static directory, this is not meant to be configurable, it is the output of a standalone program, anything using ovos-i2csound should follow the docs and check the static path, a layer of configuration here doesnt make sense to me and only introduces complexity and surface area for bugs

**Comment by JarbasAl** [comment]
i think we should use the same directory[ we use in ovos-config](https://github.com/OpenVoiceOS/ovos-config/blob/dev/ovos_config/meta.py#L125), no reason to refer to `mycroft` here when this has nothing to do with `mycroft` directly

`/etc/OpenVoiceOS/XXX`

standardizing this path as the "OS level" directory for OVOS i think makes sense, whenever we need something not tied to a user

**Comment by j1nx** [comment]
Doesn't this then also needs a 'mkdir -p /etc/OpenVoiceOS' to nake sure it exists?

File: install.sh [hunk 2]
```diff
@@ -37,6 +37,9 @@ main() {
     copy_file "$PWD/ovos-i2csound" "$script_dir/ovos-i2csound"
     chmod +x "${script_dir}/ovos-i2csound"
     systemctl enable i2c_platform.service
+    if [[ ! -d /etc/OpenVoiceOS ]]; then
+        mkdir /etc/OpenVoiceOS
+    fi
```
Comment (NeonDaniel) [inline install.sh:42]: just a note that this could be simplified to:
```shell
[ -d /etc/OpenVoiceOS ] || mkdir /etc/OpenVoiceOS
```

**Review by NeonDaniel — APPROVED** [review]

**Pull request merged**